common/app/experiments/Experiments.scala

```diff
@@ -54,3 +55,12 @@ object OphanNext
       sellByDate = LocalDate.of(2024, 4, 1),
       participationGroup = Perc1B,
     )
+
+object DCRTagFronts
```

Comment:
I'm really not too fussed about this but I think CP said we should call those tag pages. Should we consider renaming this and the `TagFrontPicker`?

Reply:
renamed ✔️